plugin/trino-pinot/src/main/java/io/trino/plugin/pinot/PinotColumnHandle.java

```diff
@@ -56,6 +64,12 @@ public ColumnMetadata getColumnMetadata()
         return new ColumnMetadata(getColumnName(), getDataType());
     }
 
+    @JsonProperty
+    public boolean isPreservePinotEmptyAggregateValue()
```

Comment:
It's not very clear what this means. I would call this "returnNullOnEmptyGroup" and adjust the code accordingly.